devops/openshift/database-backup/backup.conf

```diff
@@ -0,0 +1,12 @@
+postgres=patroni-master:5432/SIMSDB
+postgres=patroni-master:5432/sfasdb
+postgres=patroni-master:5432/ffa-camunda
+postgres=patroni-master:5432/ffa-api
+postgres=patroni-master:5432/ffa-analytics
+postgres=patroni-master:5432/ffa-session
+mongo=mongo:27017/formio
```

Comment:
for now can we remove the analytics and session db backup?

Reply:
Done!